cli/cmd/clean.go

```diff
@@ -62,47 +62,49 @@ func collectFiles(files map[string]bool, dirname string) (map[string]bool, error
 	return files, nil
 }
 
-func clean(run *running.InstanceCtx) error {
+func clean(run *running.InstanceCtx, verbose bool) (int, error) {
 	removeFiles := map[string]bool{}
 	confirm := false
 	var err error
 
 	for _, dir := range [...]string{run.LogDir, run.WalDir, run.VinylDir, run.MemtxDir} {
 		removeFiles, err = collectFiles(removeFiles, dir)
 		if err != nil {
-			return err
+			return 0, err
 		}
 	}
 
-	if len(removeFiles) == 0 {
-		log.Infof("Already cleaned.\n")
-		return nil
-	}
+	if verbose {
+		if len(removeFiles) == 0 {
+			log.Infof("Already cleaned.")
+			return 0, nil
+		}
 
-	log.Infof("List of files to delete:\n")
-	for file, _ := range removeFiles {
-		log.Infof("%s", file)
+		log.Infof("List of files to delete: ")
```

Comment:
There is no point in printing list of files to delete in the "force" mode. It is more useful to print which files were deleted is verbose mode is enabled.